ONSdigital/census-rm-acceptance-tests-2021
# Making Dockerfile Use Non-root User

# Motivation and Context
Changing the Dockerfile to use non-root user rather than root which makes it more secure (someone could gain root access to the Docker host by hacking the application running inside the container)

# What has changed
Created new user in Dockerfile and assigned permission levels. Also optimised the order in which the image is built. Made sure that any reference to original root user in app was changed to new location for non-root user. This relates to the ATs and qidbatchrunner

# How to test? 
Clone the other repos and build the Docker images and run the ATs locally. Then run them on GCP with new Docker images. Make sure to change QID_BATCH_BRANCH to branch name before runn ing ATs in GCP

# Links
https://trello.com/c/Pd48zUsd
